src/main/java/org/embulk/input/jira/client/JiraClient.java

```diff
@@ -195,13 +195,16 @@ private String extractErrorMessages(String errorResponse)
     }
 
     @VisibleForTesting
-    public HttpClient createHttpClient()
+    public CloseableHttpClient createHttpClient()
     {
-        RequestConfig config = RequestConfig.custom()
-                .setConnectTimeout(CONNECTION_TIME_OUT)
-                .setConnectionRequestTimeout(CONNECTION_TIME_OUT)
-                .build();
-        return HttpClientBuilder.create().setDefaultRequestConfig(config).build();
+        return HttpClientBuilder.create()
+                    .setDefaultRequestConfig(RequestConfig.custom()
+                                                        .setConnectTimeout(HTTP_TIMEOUT)
+                                                        .setConnectionRequestTimeout(HTTP_TIMEOUT)
+                                                        .setSocketTimeout(HTTP_TIMEOUT)
+                                                        .setCookieSpec(CookieSpecs.STANDARD)
```

Comment:
Question: what is the relation of "cookie spec" here with the current issue?

Reply:
This is due to legacy of HTTP Client of Apache. If we do not set it, we will get a logging warning in header 
```
Invalid 'expires' attribute
```